package com.ruoyi.system.domain.agentsec;

import org.apache.commons.lang3.builder.ToStringBuilder;
import org.apache.commons.lang3.builder.ToStringStyle;
import com.ruoyi.common.core.domain.BaseEntity;

/**
 * 告警规则对象 alert_rule
 */
public class AlertRule extends BaseEntity
{
    private static final long serialVersionUID = 1L;

    private String ruleId;

    private String ruleName;

    private String riskLevel;

    private String status;

    private String conditionJson;

    public String getRuleId()
    {
        return ruleId;
    }

    public void setRuleId(String ruleId)
    {
        this.ruleId = ruleId;
    }

    public String getRuleName()
    {
        return ruleName;
    }

    public void setRuleName(String ruleName)
    {
        this.ruleName = ruleName;
    }

    public String getRiskLevel()
    {
        return riskLevel;
    }

    public void setRiskLevel(String riskLevel)
    {
        this.riskLevel = riskLevel;
    }

    public String getStatus()
    {
        return status;
    }

    public void setStatus(String status)
    {
        this.status = status;
    }

    public String getConditionJson()
    {
        return conditionJson;
    }

    public void setConditionJson(String conditionJson)
    {
        this.conditionJson = conditionJson;
    }

    @Override
    public String toString()
    {
        return new ToStringBuilder(this, ToStringStyle.MULTI_LINE_STYLE)
            .append("ruleId", getRuleId())
            .append("ruleName", getRuleName())
            .append("riskLevel", getRiskLevel())
            .append("status", getStatus())
            .toString();
    }
}
